docs/docgen/docgen.md

```diff
@@ -0,0 +1,27 @@
+### Simple generation of documentation from yaml message files for Api mediation layer
```

Comment:
This is cool!

Reply:
![GitHub Logo](https://i.imgflip.com/1wxo1j.jpg)